* always use OpenSSL's MD5, never Solaris' MD5 implementation (we want
  to use whatever we provide ourself, but also not get libintl.so.1
  references from non-prefix dragged in...)
* patching of both configure and configure.in to avoid having to reconf
* don't patch configure.ac, or one gets a reconf, bug #286848

* --- configure.ac
* +++ configure.ac
* @@ -307,27 +307,6 @@
*    AC_LIBOBJ([gen-md5])
*    found_md5=no
*  
* -  dnl Check for the system MD5 library on Solaris.  We don't check for
* -  dnl something simple like "MD5Update" because there are a number of
* -  dnl MD5 implementations that use that name, but have an otherwise
* -  dnl incompatible interface.  md5_calc is, hopefully, specific to the
* -  dnl Solaris MD5 library.
* -  if test x"$found_md5" = xno; then
* -    AC_CHECK_LIB(md5, md5_calc, [
* -      dnl Some installations have bogus <md5.h> in the compiler's
* -      dnl include path, making the system md5 library useless.
* -      AC_MSG_CHECKING([for working md5.h])
* -      AC_COMPILE_IFELSE([#include <md5.h>
* -                        ], [
* -        AC_MSG_RESULT(yes)
* -        AC_DEFINE([HAVE_SOLARIS_MD5], 1, [Define when using Solaris MD5.])
* -        LIBS="-lmd5 $LIBS"
* -        found_md5=yes
* -        AC_MSG_NOTICE([using the Solaris MD5 implementation])
* -      ], [AC_MSG_RESULT(no)])
* -    ])
* -  fi
* -
*    dnl Then see if we're linking OpenSSL anyway; if yes, use its md5
*    dnl implementation.
*    if test x"$found_md5" = xno; then
--- configure
+++ configure
@@ -28059,125 +28059,6 @@
 
   found_md5=no
 
-            if test x"$found_md5" = xno; then
-    { $as_echo "$as_me:$LINENO: checking for md5_calc in -lmd5" >&5
-$as_echo_n "checking for md5_calc in -lmd5... " >&6; }
-if test "${ac_cv_lib_md5_md5_calc+set}" = set; then
-  $as_echo_n "(cached) " >&6
-else
-  ac_check_lib_save_LIBS=$LIBS
-LIBS="-lmd5  $LIBS"
-cat >conftest.$ac_ext <<_ACEOF
-/* confdefs.h.  */
-_ACEOF
-cat confdefs.h >>conftest.$ac_ext
-cat >>conftest.$ac_ext <<_ACEOF
-/* end confdefs.h.  */
-
-/* Override any GCC internal prototype to avoid an error.
-   Use char because int might match the return type of a GCC
-   builtin and then its argument prototype would still apply.  */
-#ifdef __cplusplus
-extern "C"
-#endif
-char md5_calc ();
-int
-main ()
-{
-return md5_calc ();
-  ;
-  return 0;
-}
-_ACEOF
-rm -f conftest.$ac_objext conftest$ac_exeext
-if { (ac_try="$ac_link"
-case "(($ac_try" in
-  *\"* | *\`* | *\\*) ac_try_echo=\$ac_try;;
-  *) ac_try_echo=$ac_try;;
-esac
-eval ac_try_echo="\"\$as_me:$LINENO: $ac_try_echo\""
-$as_echo "$ac_try_echo") >&5
-  (eval "$ac_link") 2>conftest.er1
-  ac_status=$?
-  grep -v '^ *+' conftest.er1 >conftest.err
-  rm -f conftest.er1
-  cat conftest.err >&5
-  $as_echo "$as_me:$LINENO: \$? = $ac_status" >&5
-  (exit $ac_status); } && {
-	 test -z "$ac_c_werror_flag" ||
-	 test ! -s conftest.err
-       } && test -s conftest$ac_exeext && {
-	 test "$cross_compiling" = yes ||
-	 $as_test_x conftest$ac_exeext
-       }; then
-  ac_cv_lib_md5_md5_calc=yes
-else
-  $as_echo "$as_me: failed program was:" >&5
-sed 's/^/| /' conftest.$ac_ext >&5
-
-	ac_cv_lib_md5_md5_calc=no
-fi
-
-rm -rf conftest.dSYM
-rm -f core conftest.err conftest.$ac_objext conftest_ipa8_conftest.oo \
-      conftest$ac_exeext conftest.$ac_ext
-LIBS=$ac_check_lib_save_LIBS
-fi
-{ $as_echo "$as_me:$LINENO: result: $ac_cv_lib_md5_md5_calc" >&5
-$as_echo "$ac_cv_lib_md5_md5_calc" >&6; }
-if test "x$ac_cv_lib_md5_md5_calc" = x""yes; then
-
-                  { $as_echo "$as_me:$LINENO: checking for working md5.h" >&5
-$as_echo_n "checking for working md5.h... " >&6; }
-      cat >conftest.$ac_ext <<_ACEOF
-#include <md5.h>
-
-_ACEOF
-rm -f conftest.$ac_objext
-if { (ac_try="$ac_compile"
-case "(($ac_try" in
-  *\"* | *\`* | *\\*) ac_try_echo=\$ac_try;;
-  *) ac_try_echo=$ac_try;;
-esac
-eval ac_try_echo="\"\$as_me:$LINENO: $ac_try_echo\""
-$as_echo "$ac_try_echo") >&5
-  (eval "$ac_compile") 2>conftest.er1
-  ac_status=$?
-  grep -v '^ *+' conftest.er1 >conftest.err
-  rm -f conftest.er1
-  cat conftest.err >&5
-  $as_echo "$as_me:$LINENO: \$? = $ac_status" >&5
-  (exit $ac_status); } && {
-	 test -z "$ac_c_werror_flag" ||
-	 test ! -s conftest.err
-       } && test -s conftest.$ac_objext; then
-
-        { $as_echo "$as_me:$LINENO: result: yes" >&5
-$as_echo "yes" >&6; }
-
-cat >>confdefs.h <<\_ACEOF
-#define HAVE_SOLARIS_MD5 1
-_ACEOF
-
-        LIBS="-lmd5 $LIBS"
-        found_md5=yes
-        { $as_echo "$as_me:$LINENO: using the Solaris MD5 implementation" >&5
-$as_echo "$as_me: using the Solaris MD5 implementation" >&6;}
-
-else
-  $as_echo "$as_me: failed program was:" >&5
-sed 's/^/| /' conftest.$ac_ext >&5
-
-	{ $as_echo "$as_me:$LINENO: result: no" >&5
-$as_echo "no" >&6; }
-fi
-
-rm -f core conftest.err conftest.$ac_objext conftest.$ac_ext
-
-fi
-
-  fi
-
       if test x"$found_md5" = xno; then
     if test x"$LIBSSL" != x; then
 
